templates/docs/patterns/hero/index.md

```diff
@@ -17,12 +17,105 @@ The hero pattern is composed of the following elements:
 
 | Element              | Description                                                              |
 | -------------------- | ------------------------------------------------------------------------ |
-| Title (**required**) | <code>h1</code> title text                                               |
-| Subtitle             | <code>h2</code> subtitle text                                            |
-| Description          | <code>p</code> description text                                          |
+| Title (**required**) | `h1` title text                                                          |
+| Subtitle             | `h2` subtitle text                                                       |
+| Description          | `p` description text                                                     |
 | Call to action block | [Call to action block](/docs/patterns/cta-block) beneath the description |
 | Image                | Main hero visual                                                         |
 
+## Jinja Macro
+
+### Parameters
+
+<div style="overflow: auto;">
+  <table>
+    <thead>
+      <tr>
+        <th style="width: 190px;">Name</th>
+        <th style="width: 100px;">Is Required</th>
+        <th style="width: 200px;">Type</th>
+        <th style="width: 150px;">Default</th>
+        <th style="width: 250px;">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>layout</code></td>
+        <td>Yes</td>
+        <td>
+          One of:
+          <ul class="p-list">
+            <li class="p-list__item has-bullet"><code>'50/50'</code></li>
+            <li class="p-list__item has-bullet"><code>'50/50-full-width-image'</code></li>
+            <li class="p-list__item has-bullet"><code>'75/25'</code></li>
+            <li class="p-list__item has-bullet"><code>'25/75'</code></li>
+            <li class="p-list__item has-bullet"><code>'fallback'</code></li>
+          </ul>
+        </td>
+        <td><code>'fallback'</code></td>
+        <td>Choice of hero layout.</td>
+      </tr>
+      <tr>
+        <td><code>title_text</code></td>
+        <td>Yes</td>
+        <td><code>string</code></td>
+        <td><code>N/A</code></td>
+        <td><code>h1</code> title text</td>
+      </tr>
+      <tr>
+        <td><code>subtitle_text</code></td>
+        <td>No</td>
+        <td><code>string</code></td>
+        <td><code>N/A</code></td>
+        <td><code>h2</code> subtitle text</td>
+      </tr>
+      <tr>
+        <td><code>is_split_on_medium</code></td>
+        <td>Yes</td>
+        <td><code>boolean</code></td>
+        <td><code>false</code></td>
+        <td>Whether the layout is split on tablet-sized devices.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+### Slots
+
+<div style="overflow: auto;">
+  <table>
+    <thead>
+      <tr>
+        <th style="width: 155px;">Name</th>
```

Comment:
Why fixed px values? We usually use % values that add up to 100%.

Reply:
Adjusted to use percentages